src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,40 +1,74 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import classNames from 'classnames';
+import { useAppDispatch } from '../../app/hooks';
 import { Loader } from '../Loader';
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
+import { getUser } from '../../api';
+import { actions } from '../../features/currentTodo';
+
+type Props = {
+  todo: Todo;
+};
+
+export const TodoModal: React.FC<Props> = ({ todo }) => {
+  const dispatch = useAppDispatch();
+  const [user, setUser] = useState<User | null>(null);
+  const { userId, id: todoId, completed } = todo;
+
+  useEffect(() => {
+    if (userId) {
+      getUser(userId).then(setUser);
+    }
+  }, []);
 
-export const TodoModal: React.FC = () => {
   return (
     <div className="modal is-active" data-cy="modal">
       <div className="modal-background" />
 
-      <Loader />
+      {!user ? (
+        <Loader />
+      ) : (
+        <div className="modal-card">
+          <header className="modal-card-head">
+            <div
+              className="modal-card-title has-text-weight-medium"
+              data-cy="modal-header"
+            >
+              Todo #
+              {todoId}
+            </div>
 
-      <div className="modal-card">
-        <header className="modal-card-head">
-          <div
-            className="modal-card-title has-text-weight-medium"
-            data-cy="modal-header"
-          >
-            Todo #3
-          </div>
-
-          {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-          <button type="button" className="delete" data-cy="modal-close" />
-        </header>
+            {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
+            <button
```

Comment:
Try to add `aria-label` and remove exception comment. it can fix the problem, and u would not need to do exception of the rule.